src/handwritten/_install_daqmx.py

```diff
@@ -323,8 +302,8 @@ def installdriver() -> None:
             logging.info("Windows platform detected")
             _install_daqmx_windows_driver()
         else:
-            print(f"installdriver command is supported only on Windows")
+            raise click.ClickException(f'Error installdriver command is supported only on Windows OS.')
```

Comment:
Don't print "Error" without a colon because that's confusing. Also, `ClickException` should already prefix the error message with "Error:".